doc/publishing-application-examples.md

```diff
@@ -0,0 +1,194 @@
+                           Responds with version                       |
+                    <--------------------------------------------------
+
+  [ Updated draft ] -----------------------------------> [ PUT /v2/content/:content_id ]
+          |
+          |            Publish request
+           --------------------------------------------> [ POST /v2/content/:content_id/publish ]
+```
+
+## Configuring the Publishing API for use in a publishing application
+
+HTTP requests to Publishing API endpoints should be made using the [publishing-api v2 gds-api-adapters client library](https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api/publishing_api_v2.rb).
+A trivial example of how to configure a publishing application to use the publishing API v2 client adapter would be
+
+```
+  require "gds_api/publishing_api_v2"
+
+  publishing_api = GdsApi::PublishingApiV2.new(Plek.new("publishing-api"))
+
+```
+
+## Drafting a content item from a publishing app
+
+Publishing applications should use the `PUT /v2/content/:content_id` endpoint to add content.
+Please refer to the [syntactic documentation]() and [semantic documentation]() for further details.
+
+The publishing application is responsible for the generation of a content_id, the primary identifier for content in the publishing API and content store. The convention is to use SecureRandom.uuid to generate a valid content_id.
+
+Using the publishing API v2 client adapter with a valid content_id and payload, the following example would make the request.
+
+```
+  publishing_api = GdsApi::PublishingApiV2.new(Plek.new("publishing-api"))
+  content_id = SecureRandom.uuid
+  guide = GuideContentModel.new(
+    content_id: content_id,
+    base_path: "/vat-rates",
+    title: "VAT rates",
+    description: "VAT rates for goods and services",
+    details: { body: "Something about VAT" },
+    access_limited : ["8f3173bb-ee2a-48a2-aa86-f2f030830888"],
+    rendering_app: "frontend",
+    publishing_app: "my-shiny-publishing-app",
+    public_updated_at: "2014-05-14T13:00:06.000Z",
+    format: "guide",
+    routes: [{
+      path: "/vat-rates", type: "exact"
+    }]
+  )
+  payload = guide.as_json
+  publishing_api.put_content(content_id, payload)
+```
+
+
+The response body would contain a presentation of the saved content item including the item version eg.
+
+```
+  {"id":1,
+  "content_id":"940b88db-8f15-4859-b5b2-4761ba62a067",
+  "locale":"en",
+  "base_path":"/vat-rates",
+  "title":"VAT rates",
+  "description":"VAT rates for goods and services",
+  "format":"guide",
+  "public_updated_at":"2014-05-14T13:00:06.000Z",
+  "details":{"body":"Something about VAT"},
+  "access_limited":["8f3173bb-ee2a-48a2-aa86-f2f030830888"],
+  "routes":[{"path":"/vat-rates","type":"exact"}],
+  "redirects":[],
+  "publishing_app":"my-shiny-publishing-app",
+  "rendering_app":"frontend",
+  "version":"1"}
+```
+
+which could then be used to update the model instance:
```

Comment:
Just to avoid any possible confusion I'd add a word "local" e.g. 
> which could then be used to update the local model instance in the publishing application: